### ISSUE TITLE
Carousel (landing and about page)

### ISSUE THREAD

**Issue by rebeccacremona** [harvard-lil/perma#2240]
From Bootstrap: 
>Accessibility issue
>
>The carousel component is generally not compliant with accessibility standards. If you need to be compliant, please consider other options for presenting your content.

This library is usually pretty good, and has a patch for at least partially accessibility: https://github.com/paypal/bootstrap-accessibility-plugin. Look into that, and see how it goes. Maybe we can get it good enough?

The controls for the carousel should be buttons, not links, and they need to be labeled.
https://github.com/harvard-lil/perma/blob/develop/perma_web/perma/templates/includes/participating_institutions.html#L19

Also the "carousel indicators" are just `<li>` right now; they should be buttons

(maybe we can rip out this whole thing......)

**Comment by rebeccacremona** [comment]
We're going to remove the carousel. See https://github.com/harvard-lil/perma/issues/2315
